src/ios/AssetBundleDownloader.swift

```diff
@@ -308,7 +308,15 @@ final class AssetBundleDownloader: NSObject, URLSessionDelegate, URLSessionTaskD
       }
 
       do {
-        try fileManager.moveItem(at: location, to: asset.fileURL as URL)
```

Comment:
Can we just catch the error if this fails, instead of trying to anticipate whether it's going to fail?